frontend/src/App.css

```diff
@@ -1,6 +1,3 @@
 #root {
-  max-width: 1280px;
-  margin: 0 auto;
-  padding: 2rem;
-  text-align: center;
+ font-family: 'BC Sans',
```

Comment:
Should declare a secondary font as a backup if BCSans fails
https://www.w3schools.com/css/css_font_fallbacks.asp

Reply:
added sans-serif, though on header-style